### DISCUSSION TITLE
Data Quality Rules for Draft Object

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17511] [Q&A / Support]
Is it possible to write Data Quality rules that run against the draft object? For example if I want to use the Data Quality rule as a guard for a workflow transition, I would like to evaluate against the draft object as I don't publish the object until the end of the workflow.

**Comment by fashxp** [comment]
As far as I understand, this should be possible, yes. 
When you just save a draft version, the score calculation picks up the latest draft version, calculates the scores based on it and stores it again. 

![image](https://github.com/user-attachments/assets/1108db63-e368-4228-b795-0991d55968df)
